components/blog/Categories.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <b-row
+    class="flex-md-row"
+    data-qa="blog categories"
+  >
+    <b-col cols="12">
+      <h3 class="font-weight-bold">
+        #{{ categoryTitle }}
+      </h3>
+      <ul class="list-unstyled">
+        <li
+          v-for="(category) in categories"
+          :key="category"
+        >
+          <SmartLink
+            :destination="category"
+          >
+            {{ category }}
+          </SmartLink>
+        </li>
+      </ul>
+    </b-col>
+  </b-row>
+</template>
+
+<script>
+  import SmartLink from '../generic/SmartLink';
+
+  export default {
+    name: 'Categories',
```

Comment:
[Component names should always be multi-word.](https://vuejs.org/v2/style-guide/#Multi-word-component-names-essential)